scss/dialog/_theme.scss

```diff
@@ -5,15 +5,10 @@
 
 
     // Titlebar
-    .k-dialog-titlebar {
```

Comment:
Ahh - I was misinformed then.  

Will correct and introduce some new variables.